src/main/java/picocli/CommandLine.java

```diff
@@ -12377,6 +12437,22 @@ private void processArguments(List<CommandLine> parsedCommands,
                 // A single option may be without option parameters, like "-v" or "--verbose" (a boolean value),
                 // or an option may have one or more option parameters.
                 // A parameter may be attached to the option.
+                if (commandSpec.parser().abbreviatedOptionsAllowed() && arg.startsWith("--")) {
```

Comment:
Sometimes long (non-POSIX) options can start with a single hyphen `-`, or some other prefix (picocli allows any prefix). Perhaps we can think of some other way to exclude POSIX options.

Reply:
We might enforce the scope of options abbreviation to ones that start with dual hyphens? Otherwise it will inevitably cause ambiguity.

Or we can note in the document that abbreviation only works fully when `setPosixClusteredShortOptionsAllowed(false)`.

Another way is to provide an option for the users to choose what they want...

Reply:
Or make an `abbreviatedTransformer`